src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -934,6 +945,48 @@ class DukFromGameActionParameterVisitor : public GameActionParameterVisitor
     }
 };
 
+const static std::unordered_map<std::string, uint32_t> ActionNameToType = {
+    { "guestsetname", GAME_COMMAND_SET_GUEST_NAME },
+    { "parksetname", GAME_COMMAND_SET_PARK_NAME },
+    { "ridecreate", GAME_COMMAND_CREATE_RIDE },
+    { "ridedemolish", GAME_COMMAND_DEMOLISH_RIDE },
+    { "rideentranceexitplace", GAME_COMMAND_PLACE_RIDE_ENTRANCE_OR_EXIT },
+    { "rideentranceexitremove", GAME_COMMAND_REMOVE_RIDE_ENTRANCE_OR_EXIT },
+    { "ridesetappearance", GAME_COMMAND_SET_RIDE_APPEARANCE },
+    { "ridesetcolourscheme.hpp", GAME_COMMAND_SET_COLOUR_SCHEME },
+    { "ridesetname", GAME_COMMAND_SET_RIDE_NAME },
+    { "ridesetprice", GAME_COMMAND_SET_RIDE_PRICE },
+    { "ridesetsetting", GAME_COMMAND_SET_RIDE_SETTING },
+    { "ridesetstatus", GAME_COMMAND_SET_RIDE_STATUS },
+    { "ridesetvehicles", GAME_COMMAND_SET_RIDE_VEHICLES },
+    { "smallsceneryplace", GAME_COMMAND_PLACE_SCENERY },
+    { "trackdesign", GAME_COMMAND_PLACE_TRACK_DESIGN },
+    { "trackplace", GAME_COMMAND_PLACE_TRACK },
+    { "trackremove", GAME_COMMAND_REMOVE_TRACK },
+    { "tracksetbrakespeed", GAME_COMMAND_SET_BRAKES_SPEED },
+};
+
+static std::string GetActionName(uint32_t commandId)
+{
+    auto it = std::find_if(
+        ActionNameToType.begin(), ActionNameToType.end(), [commandId](const auto& kvp) { return kvp.second == commandId; });
```

Comment:
`kvp` ?  Can we use a more meaningful name? (or is it an acronym I missed?)

Reply:
"key value pair", pretty common naming convention isn't it?

Reply:
Seems alright